openassessment/xblock/config_mixin.py

```diff
@@ -12,13 +12,11 @@
 WAFFLE_NAMESPACE = 'openresponseassessment'
 
 ALL_FILES_URLS = 'all_files_urls'
-MOBILE_SUPPORT = 'mobile_support'
 TEAM_SUBMISSIONS = 'team_submissions'
 USER_STATE_UPLOAD_DATA = 'user_state_upload_data'
 
 FEATURE_TOGGLES_BY_FLAG_NAME = {
     ALL_FILES_URLS: 'ENABLE_ORA_ALL_FILE_URLS',
-    MOBILE_SUPPORT: 'ENABLE_ORA_MOBILE_SUPPORT',
```

Comment:
@iloveagent57: I think we have this waffle turned on in Stage, although we also have `FEATURES['ENABLE_ORA_MOBILE_SUPPORT']` turned on in Stage.  Do you agree that waffle/switch should not be supported for this toggle, and would you mind deleting the flag `ENABLE_ORA_MOBILE_SUPPORT` in Stage?